samples/bluetooth/bap_broadcast_sink/src/main.c

```diff
@@ -606,118 +608,66 @@ static struct bt_bap_stream_ops stream_ops = {
 };
 
 #if defined(CONFIG_TARGET_BROADCAST_CHANNEL)
-struct find_valid_bis_data {
+struct bis_channel_allocation_data {
 	struct {
 		uint8_t index;
 		enum bt_audio_location chan_allocation;
+		bool chan_allocation_available;
```

Comment:
Suggest to move the bool above `chan_allocation` or `index` below `chan_allocation` so we don't have `1 | 4 | 1` in memory

Reply:
Ok. Good point